Libplanet/Net/Swarm.cs

```diff
@@ -182,7 +181,11 @@ static Swarm()
 
         public Peer AsPeer => Transport.AsPeer;
 
-        public DateTimeOffset LastReceived { get; private set; }
+        /// <summary>
+        /// The time the message was last received.
+        /// </summary>
+        public DateTimeOffset? LastMessageTimestamp =>
+            Running ? Transport.LastMessageTimestamp : default(DateTimeOffset?);
```

Comment:
Since it's nullable, how about using `null`?
```suggestion
            Running ? Transport.LastMessageTimestamp : (DateTimeOffset?)null;
```

Reply:
I guess `default(DateTimeOffset?)` and `(DateTImeOffset?)null` are same. (both are `null`).

Reply:
You're right. I thought that `null` would be a bit more clear, but it is not a strong opinion.

Reply:
I find `default(DateTimeOffset?)` prone to be misread as `default(DateTimeOffset)` as well…

Reply:
I see, I'll fix it.

Reply:
Amended. a0bf5ddeab14721822bf02af1d5529f2b30c5d98